### ISSUE TITLE
"count" problem with inner join in v4.5.1+

### ISSUE THREAD

**Issue by katatunix** [Dzoukr/Dapper.FSharp#92]
I believe from v4.5.1 (and currently v4.6.0) the lib has problem with `count`  and `innerJoin`. For example:

```fsharp
select {
    for p in table'<{| Id: string; GameId: string |}> "Players" do
    innerJoin g in table'<{| Id: string |}> "Games" on (p.GameId = g.Id)
    count "*" "Value"
} |> conn.Select<{|Value: int64|}>
```

I used the `Deconstructor` to see the raw SQL query:

```sql
SELECT `Players`.`Value` FROM `Players` INNER JOIN Games ON `Games`.`Id`=`Players`.`GameId`
```

Indeed the SQL query did not contain the `count(*)` statement.

Note:
- Without `innerJoin` it still works well.
- In v4.5.0 and before, everything works well.
- I'm using MySQL v8.0.

Hope you will consider this.

**Comment by Dzoukr** [comment]
Oh, thanks for letting me know - will look at it!

**Comment by Dzoukr** [comment]
Fixed in 4.6.1, thanks to @jindraivanek ❤️
